docs/standard/changelog.rst

```diff
@@ -6,25 +6,66 @@ ChangeLog
 
 .. include:: warningbox.rst
 
-[Unreleased]
-============
+0.4 - 2024-05-04
+================ 
+
+Added
+-----
+* Separation of metadata and data into statements and records
+* ``declaration`` and ``declarationSubject`` to model declarations
+* ``recordId`` and ``recordStatus`` to model change over time 
+* New documentation pages: 
+
+  * Dates guidance 
+  * Generating statements
+  * Record identifiers
+  * Representing trusts 
+  * Representing nominees 
+  * Representing record updates
 
 Changed
 -------
-- Removed ``securitiesListing`` from required fields in ``hasPublicListing``.
-- Scenario added to 'Representing state-owned enterprises' page covering ownership or control by a regional government.
+* valid date formats for all date fields restricted 
+* Schema and codelist titles and descriptions updated
+* Documentation images and text updated 
+* Documentation guidance restructured into modelling requirements and system requirements
+* Ownership-or-control statements replaced with Relationship statements
+* Statement fields:
+  * ``statementDate`` now required
+  * ``statementType`` replaced with ``recordType``
+  * ``statementID`` renamed ``statementId``
+* Entity fields: 
+  * ``entityType`` and ``entitySubtype`` combined into ``entityType``
+  * ``securitiesListing`` removed from required fields in ``hasPublicListing``
+  * ``nomination`` and ``trust`` added to ``entitySubtype`` codelist
+  * prefixes removed from ``entitySubtype`` codelist (e.g. ``stateBody-stateAgency`` is now ``stateAgency``) 
+* Relationship fields:
+  * ``Interest/share/maximum`` is now inclusive by default. ``exclusiveMinimum`` and ``exclusiveMaximum`` are now numbers
+  * ``nominee`` and ``nominator`` added to interest type codelist 
+  * ``interestedParty.describedByEntityStatement``, ``interestedParty.describedByPersonStatement``, ``interestedParty.unspecified`` combined into ``interestedParty`` which is either a string referring to a recordId or an Unspecified Record. 
```

Comment:
```suggestion
  * ``interestedParty`` and ``subject`` fields now hold a ``recordId`` value (rather than a ``statementId`` value). They may instead take an Unspecified Record object, to represent missing information.
```